hal/halx86/acpi/busemul.c

```diff
@@ -193,9 +200,7 @@ HalGetBusDataByOffset(IN BUS_DATA_TYPE BusDataType,
                       IN ULONG Offset,
                       IN ULONG Length)
 {
-    BUS_HANDLER BusHandler;
-
-    /* Look as the bus type */
+    /* Look at the bus type */
     if (BusDataType == Cmos)
```

Comment:
Please exclude the CMOS bus support from the minihal library using ifdef guards.